src/tests/api/BavHappyPath.test.ts

```diff
@@ -16,22 +17,27 @@ describe("Test BAV End Points", () => {
 
     beforeEach(async () => {
         //Session Request
-        sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
+       sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
     });
-    it("E2E BAV End Points Happy Path Journey", async () => {
+    it.skip("E2E BAV End Points Happy Path Journey", async () => {
```

Comment:
Why has the /session validation been removed? 

We should not be removing any existing logic and adding to it instead. 

Please re-add the authSessionState validation that was part of the /session endpoint PR and then you should kick off your logic to hit the /authorization endpoint and then validate the authSessionState again